Minor edits to SC CLI, YAML config (#1041)
saucelabs/sauce-docs: docs/dev/cli/sauce-connect-proxy.md

<p><small>| REQUIRED | STRING |</small></p>

:::caution For YAML Configuration Files ONLY
This is required only if you're using a [YAML file to configure your tunnel](/secure-connections/sauce-connect/setup-configuration/yaml-config/).
:::

__Description__: Defines the local path to a YAML file containing a Sauce Connect Proxy configuration..<br/>
__Shorthand__: `-c`

---
### `--region`
<p><small>| OPTIONAL | STRING |</small></p>

> __Description__: Sets your Sauce Labs regional data center such as `us-west` and `eu-central`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>

Reviewer: maybe "such as `us-west` or `eu-central`"

> __Description__: Sets your Sauce Labs regional data center such as `us-west` and `eu-central`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
> __Default__: If you don't specify a data center, it will default to `us-west`.<br/>

Reviewer 2: ```suggestion
__Description__: Sets your Sauce Labs regional data center, such as `us-west` or `eu-central`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
__Default__: Defaults to `us-west` when not specified.<br/>
```

### `--tunnel-domains`
<p><small>| OPTIONAL | STRING |</small></p>

__Description__: Sets domain(s) that need to be sent through the Sauce Connect Proxy tunnel. This is the inverse of `--direct-domains`.  When adding multiple domains, [format as a comma-separated list](#formatting-domains-in-the-command-line).<br/>
__Shorthand__: `-t`

---
### `--rest-url`
<p><small>| OPTIONAL | STRING |</small></p>

> __Description__: Sets your [Sauce Labs regional data center REST API URL](#data-center-endpoints) (e.g., EU-Central, US-West).<br/>

Reviewer: I think should remove `(e.g., EU-Central, US-West).`

### `--ocsp`
<p><small>| OPTIONAL | STRING |</small></p>

__Description__: Sets OCSP verification mode. Options are: `strict`, `log-only`, and `disable`.<br/>

> __Default__: log-only<br/>

Reviewer 2: ```suggestion
__Default__: Defaults to `log-only` when not specified.<br/>
```
